test/quantization/eager/test_quantize_eager_ptq.py

```diff
@@ -1444,26 +1443,6 @@ def forward(self, indices, offsets, linear_in):
         self.assertTrue('DynamicQuantizedLinear' in str(q_model))
 
 class TestQuantizeEagerONNXExport(JitTestCase):
```

Comment:
this class has onnx name in there somehow..

Reply:
Yes, but other teams are using it, we should coordinate the name change with them